src/api/app/views/webui/kiwi/images/show.html.haml

```diff
@@ -1,27 +1,122 @@
 - @layouttype = 'custom'
 
-.grid_16.alpha.omega.box.box-shadow
-  = render partial: 'webui/kiwi/tabs'
-  %h2
-    = @image.name
-  %ul.horizontal-list
-    %li
-      = link_to(sprited_text('brick_edit', 'Edit image'), edit_kiwi_image_path(@image))
-
-.grid_16.alpha.omega.box.box-shadow
-  %h3 Repositories
-  - if @image.repositories.empty?
-    %p There are no repositories.
-  - else
-    %ul
-      - @image.repositories.each do |repository|
-        = render partial: 'repository_entry', locals: {repository: repository}
-
-.grid_16.alpha.omega.box.box-shadow
-  %h3 Packages
-  - if @image.kiwi_packages.empty?
-    %p There are no packages.
-  - else
-    %p
-      %strong
-        = @image.kiwi_packages.pluck(:name).join(', ')
+= form_for @image, html: { id: 'kiwi-image-update-form' } do |f|
+
+  .grid_16.alpha.omega.box.box-shadow
+    = render partial: 'webui/kiwi/tabs'
+    %h2
+      = @image.name
+    %ul.horizontal-list
+      %li
+        = button_tag('Save', id: 'kiwi-image-update-form-save', type: 'button', disabled: true)
```

Comment:
Probably not, because this break all the user workflow. It should be down. But I would suggest making this change in a different PR, just to avoid making this PR bigger, it is already huge. :thinking: